x/bank/types/keys.go

```diff
@@ -37,13 +37,6 @@ var (
 	ParamsKey = []byte{0x05}
 )
 
-const (
```

Comment:
IDK why these were added. Gogo already has wrappers for native types. Let's not add more code that's not needed.